src/components/Campaigns/SingleCampaign/index.tsx

```diff
@@ -7,8 +7,18 @@ import { useSingleCampaignHandler } from "../../../hooks/useSingleCampaignHandle
 import { Link } from "react-router-dom";
 
 function SingleCampaign() {
-  const { campaign, mediaLinkIdx, prevItem, nextItem, campaigns, allStages } =
-    useSingleCampaignHandler();
+  const {
+    campaign,
+    mediaLinkIdx,
+    prevItem,
```

Comment:
We actually do not need pagination feature involved, it's just the buttons that we need. 1 through 50+. Please revert the pagination functionality but just keep the buttons. It would be nice to see a screenshot of it on the PR.